Sources/SwiftDown/SwiftDownViewer.swift

```diff
@@ -14,21 +14,21 @@ typealias ViewRepresentable = NSViewRepresentable
 import Down
 
 struct MarkdownRepresentable: ViewRepresentable {
-    @Binding var markdown: String
+    var markdown: String
 
 #if os(iOS)
     func makeUIView(context: Context) -> DownTextView {
         let styler = loadDefaultDownStyler(context.environment.colorScheme)
         let downView = DownTextView(frame: .zero, styler: styler)
         downView.text = self.markdown
-        downView.textAlignment = .left
-        downView.isScrollEnabled = false
+        downView.textAlignment = NSTextAlignment.left
+        downView.isScrollEnabled = true
```

Comment:
does this mean we scroll in the view now and not in the whole app? like within the markdown box rather than the whole app?

Reply:
nvm this code is anyways only for the markdown editor.

Reply:
Good to keep this in mind; it solved the dissapearing of parts of the note to enable scrolling here (compared to in the SwiftDownViewer), but issues might arise when we put this whole thing in a ScrollView somewhere in the app.